src/components/FindWallet/WalletTable.tsx

```diff
@@ -1,6 +1,7 @@
 // Libraries
-import React, { useState } from "react"
+import React, { useState, SVGProps } from "react"
 import { getImage, GatsbyImage } from "gatsby-plugin-image"
+import { keyframes } from "@emotion/react"
```

Comment:
Should we decide on a convention of how we animate (maybe framer-motion?) on ethereum.org?

We now have at least three ways to animate:

- motion components
- CSS animations
- keyframes

Reply:
Agree. I know that we use framer for a lot of other animations so will refactor to that.